storage/innobase/include/page0zip.ic

```diff
@@ -170,7 +170,7 @@ page_zip_rec_needs_ext(
 				ignored if zip_size == 0 */
 	ulint	zip_size)	/*!< in: compressed page size in bytes, or 0 */
 {
-	ut_ad(rec_size > comp ? REC_N_NEW_EXTRA_BYTES : REC_N_OLD_EXTRA_BYTES);
```

Comment:
What was the warning here? The issue here seems to be > operator having higher precedence than ?: one, not that either of REC_N_... constants must be non-zero

Reply:
You are right, that makes more sense I changed it.